vllm/model_executor/models/jamba.py

```diff
@@ -228,25 +229,39 @@ def forward(
         conv_state: torch.Tensor,
         ssm_state: torch.Tensor,
     ):
+        offset = 0
         if attn_metadata.prefill_metadata is not None:
-            offset = 0
-            for i, prompt_len in enumerate(
+            # seq_len = computed len + new prompt len;
+            # context_len = computed len
```

Comment:
You could borrow the below figure and nomenclature from the `backends/flash_attn.py` here
```
    # NOTE(sang): Definition of context_len, query_len, and seq_len.
    # |---------- N-1 iteration --------|
    # |---------------- N iteration ---------------------|
    # |- tokenA -|......................|-- newTokens ---|
    # |---------- context_len ----------|
    # |-------------------- seq_len ---------------------|
    #                                   |-- query_len ---|
```
The figure is a nice one to explain what's going on, and if we rename `prompt_len` -> `query_len` it'd be good for consistency  (cc @rkooo567 to doublecheck me on this)

Reply:
Great idea, added it.

Reply:
Did you mean to back this out?

Reply:
I think it's unnecessary after the varlen change

Reply:
Yeah, I've put it back